module/batch-action/src/Domain/Repository/BatchActionRepositoryInterface.php

```diff
@@ -11,12 +11,17 @@
 use Ergonode\BatchAction\Domain\Entity\BatchActionId;
 use Ergonode\BatchAction\Domain\Entity\BatchAction;
 use Ergonode\SharedKernel\Domain\AbstractId;
+use Ergonode\SharedKernel\Domain\AggregateId;
 
 interface BatchActionRepositoryInterface
 {
     public function load(BatchActionId $id): ?BatchAction;
 
     public function save(BatchAction $batchAction): void;
 
-    public function addEntry(BatchActionId $id, AbstractId $entryId): void;
+    public function addResource(BatchActionId $id, AbstractId $entryId): void;
+
+    public function markResourceAsSuccess(BatchActionId $id, AggregateId $resourceId): void;
```

Comment:
I think we are making an entry a success, not a resource itself... Maybe we need to synchronize once more for the naming convention as it seems to be troubling a bit.